### PR TITLE
Add link to glossary on english page

### PR DESCRIPTION
**Pull request by larswmh**
As a response to [this issue](https://github.com/corona-warn-app/cwa-website/issues/1625), here's the pull request to add a link to the glossary on top of the english FAQ page. 
image

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -2084,6 +2084,7 @@
     "glossary": {
         "glossaryHeadline": "Glossary",
         "optionalText": "",
+        "linkToGlossary": "Link to the glossary",
```
Comment (MikeMcC399) [inline src/data/faq.json:2087]:
```suggestion
        "linkToGlossary": "Link to the Glossary",
```
Capitalize, since it is a section heading.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Suggest to capitalize "Glossary".

File: src/data/faq.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq.json]: Changed it, thanks!

**Comment by dsarkar** [comment]
@MikeMcC399 @larswmh Thanks.

**Pull request merged**